htmd/parameterization/cli.py

```diff
@@ -497,13 +497,29 @@ def _printEnergies(molecule, parameters, filename):
            ELEC_ENERGY=energies['elec'],
            VDW_ENERGY=energies['vdw'])
 
+    for l in string.split('\n'):
+        logger.info(l)
     with open(filename, 'w') as file_:
         file_.write(string)
     logger.info('Write energy file: {}'.format(filename))
 
-    logger.info('Diagnostic energies:')
-    for name in ('bond', 'angle', 'dihedral', 'improper', 'elec', 'vdw'):
-        logger.info('   {:8s} : {:10.3f} kcal/mol'.format(name, energies[name]))
+
+def _printArguments(args, filename=None):
+
+    if filename:
```

Comment:
Looks a bit over-engineered way to write a text file.

Reply:
I've added some comments to make it more obvious